hooks/useCreateData.tsx

```diff
@@ -12,12 +13,25 @@ import handleTxError from "@/lib/handleTxError"
 import { useSheetRenderer } from "@/providers/SheetRendererProvider"
 import getTotalSupply from "@/lib/getTotalSupply"
 import usePreparePrivyWallet from "./usePrepareWallet"
+import { ethers } from "ethers"
+
+const whitelistedAddresses = [
+  "0x2d9cBC4ECFBd1B8F66Aa798FD51585AE058dAA8B",
+  "0x7A631B4a6b1D0a6b1e433d92690d7C8Aba4F23b4",
+  "0xD6C09962E907428112069273d5C0dc861e7B1C57",
+  "0x5a5811E0A22695FEC271fe908E2a1A64Dc3b06F9",
+  "0x254768D47Cf8958a68242ce5AA1aDB401E1feF2B",
+  "0xcfBf34d385EA2d5Eb947063b67eA226dcDA3DC38",
+  "0x99Fc221482ca78664d0288FfC6531cb6dfEB0c5a",
+  "0xA0EA34448738357e0c2e58147b5719A19022ac76",
+]
```

Comment:
can you move this list to the `const` file?

Reply:
Added this comment changes to the PR.